angularjs-portal-home/src/main/webapp/js/web-config.js

```diff
@@ -18,7 +18,8 @@ define(['angular'], function(angular) {
         "googleSearchURL" : "/web/api/uwrfsearch?key=AIzaSyCVAXiUzRYsML1Pv6RwSG1gunmMikTzQqY&rsz=10&num=10&hl=en&prettyPrint=false&source=gcsc&gss=.com&sig=432dd570d1a386253361f581254f9ca1&cx=004071655910512460416:8kmve-tofw8&googlehost=www.google.com&nocache=1456777578251&",
         "webSearchURL" : "https://www.uwrf.edu/AboutUs/SearchResults.cfm?q=",
         "domainResultsLabel" : "UWRF.edu",
-        'helpdeskURL' : 'https://kb.wisc.edu/helpdesk/'
+        'eventsSearchURL' : "https://events.uwrf.edu?q=",
+        'helpdeskURL' : "https://technology.uwrf.edu/TDClient/Home/"
```

Comment:
Good change, but intended in scope for this commit? Pick up another JIRA for free?